models/schemas/levelSchema.ts

```diff
@@ -245,8 +245,10 @@ export async function calcPlayAttempts(levelId: ObjectId, options: any = {}) {
   ], options);
 
   // get array of unique userIds from playattempt calc_playattempts_unique_users
+  // TODO: need to filter out play attempts where startTime === endTime
```

Comment:
need to find an efficient way to do this

also i feel like there could be some edge case where start and end time is the same for a really short level...
or maybe a user is one step away from the end, leaves their computer for over 15 min, comes back and makes one move to finish the level (JUST_BEATEN playattempt with probably equal start and end time)